.github/workflows/gcp.yml

```diff
@@ -55,3 +55,12 @@ jobs:
 
       - name: Build and Deploy Development version
         run: ./mvnw -B compile -P develop -P frontend jib:build -Djib.container.environment=SPRING_PROFILES_ACTIVE=develop -Djib.to.image=gcr.io/flock-eco/flock-eco-workday-develop --file pom.xml
+
+      - id: build-and-push-oathkeeper
+        name: Build and push Oathkeeper
+        uses: docker/build-push-action@v3
+        with:
+          context: deployments/oathkeeper
+          push: true
+          tags: eu.gcr.io/flock-eco/flock-eco-workday-oathkeeper:develop
```

Comment:
Do we wanna go 'eu' rather the us? Workday is send to `gcr.io`. I like the `eu.gcr.io` semantic, but not sure what we want here?